binancial/features/__init__.py

```
@@ -2,4 +2,4 @@
 from .is_day_highlow import is_day_highlow
 from .price_changes_related import price_changes_related
 from .time_windows import time_windows
-from .trend_count import trend_count
\ No newline at end of file
+from .trend_and_momentun import trend_and_momentun
```

Ruff (0.8.2)

`.trend_and_momentun.trend_and_momentun` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)